itou/templates/apply/includes/job_applications_filters/top_filters.html

```diff
@@ -0,0 +1,27 @@
+{% load django_bootstrap5 %}
+
+<div class="s-section__row row">
+    <div class="col-12">
+        <form hx-get="{% if request.user.is_prescriber %}{% url 'apply:list_prescriptions' %}{% elif request.user.is_employer %}{% url 'apply:list_for_siae' %}{% else %}{% url "apply:list_for_job_seeker" %}{% endif %}"
```

Comment:
Il ne faudrait pas utiliser `job_applications_list_kind is JobApplicationsListKind.SENT` & co maintenant ?

Reply:
Si, tout à fait ! Bien vu.

Reply:
J’ai ajouté un fixup avec un test.